air/src/preparation_step/errors.rs

```diff
@@ -33,9 +33,10 @@ pub enum PreparationError {
 
     /// Errors occurred on executed trace deserialization.
     #[error(
-        "an error occurred while executed trace deserialization: {0:?}.\
-    Probably it's a data of an old version that can't be converted to '{}'.\n\
-    Data: {1:?}",
+        "an error occurred while executed trace deserialization: {0:?}.\n\
+        Interpreter of {} version is used and it's expecting a data of {} version.\n\
+        Data: {1:?}",
+        super::interpreter_version(),
```

Comment:
wdyt about putting `interpreter_version` into data? So that we can say "got data from interpreter X.X.X, support minimum Y.Y.Y, current is Z.Z.Z"?

Reply:
This error was split into two: the first when versions weren't possible to recover from data and the second when it was possible.